java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -21,6 +21,7 @@ public class TransactionTestUtilities {
     private static final String listKey3 = "{key}:listKey3-" + UUID.randomUUID();
     private static final String key7 = "{key}" + UUID.randomUUID();
     private static final String key8 = "{key}" + UUID.randomUUID();
+    private static final String zsetKey2 = "{key}:zsetKey2-" + UUID.randomUUID();
```

Comment:
nit: Move so that it doesn't split key8 from key9

Reply:
Oh the diff didn't show what was above listKey3. It'd be nice and would look more organized if you could move it too. Not a blocker though.